rpc/core/env.go

```diff
@@ -58,6 +58,12 @@ type peers interface {
 	Peers() p2p.IPeerSet
 }
 
+type peerManager interface {
+	Peers() []types.NodeID
+	Add(p2p.NodeAddress) (bool, error)
```

Comment:
Looks like we don't use this `Add` method in the `rpc/core` code, can be removed